src/content/learn/rendering-lists.md

```diff
@@ -1131,7 +1141,7 @@ hr {
 
 </Sandpack>
 
-(시의 행은 절대 재 정렬되지 않기 때문에 인덱스를 key로 사용할 수 있는 드문 케이스입니다.)
+(시의 행은 절대로 순서가 바뀌지 않으므로 인덱스를 key로 사용할 수 있는 경우는 드뭅니다.)
```

Comment:
```suggestion
(시의 행은 절대로 순서가 바뀌지 않으므로 인덱스를 key로 사용할 수 있는 드문 예입니다.)
```

원문
```
(This is a rare case where index as a key is acceptable because a poem's lines will never reorder.)
```

key에 index를 넣는것은 안티패턴이지만, 이 경우에만 index를 사용해도 안전한 케이스에 대해서 설명합니다. 원문의 의미를 더 잘 전달하기 위해서 위와 같이 수정을 제안합니다.